demo/src/App.vue

```diff
@@ -220,6 +236,44 @@ onMounted(() => {
   plugin.draw('root');
   updateComponentsIds();
 });
+
+/**
+ *
```

Comment:
If you want to enable eslint in App.vue, so fix all eslint error properly or not enable it.

Reply:
What would be the proper way, if not this one, to allow console.log (ie. disable no-console eslint rule) for a single file ?

Reply:
done